src/Pages/home/home.tsx

```diff
@@ -1,8 +1,155 @@
+    const newIndex = isLastSlide ? 0 : currentIndex + 1;
+    setCurrentIndex(newIndex);
+  };
+
+  // Background images slide
+  const data = [
+    {
+      img: HomeImg,
+      label: "Go places you've dreamed of.",
+      description:
+        "Wafer dessert danish. Powder toffee cookie jelly beans bear claw jelly gingerbread halvah. and ",
+      temp: "",
+    },
+    {
+      img: CoralIsland,
+      label: "The Coral Island",
+      description: "The North Lake Iceland",
+      temp: "-16 ° C Very cold",
+    },
+    {
+      img: CrystalCaves,
+      label: "The Crystal Caves",
+      description: "The North Lake Iceland",
+      temp: "-16 ° C Very cold",
+    },
+  ];
+
+  interface ImgSlideStyle {
+    backgroundImage: string;
+    backgroundSize: string;
+    backgroundRepeat: string;
+    backgroundPosition: string;
+    width: string;
+    height: string;
+  }
+
+  const imgSlide: ImgSlideStyle = {
+    backgroundImage: `url(${data[currentIndex].img})`,
+    backgroundSize: "cover",
+    backgroundRepeat: "no-repeat",
+    backgroundPosition: "center",
+    width: "auto",
+    height: "100vh",
+  };
+
+  // Button styles
+  const buttonStyle: string =
+    "flex justify-center items-center rounded-full text-2xl w-14 h-14 bg-white cursor-pointer";
+
+  const playButtonStyle: string =
+    "font-openSan text-base text-white font-bold flex flex-row items-center w-auto h-11 cursor-pointer sm:mr-16 mr-0";
+
+  const playIconStyle: string = "text-base p-4 mr-6 rounded-full bg-red-700";
+
+  const moreInfoStyle: string =
+    "font-openSan font-bold text-white text-sm border-2 border-white border-solid w-48 h-11 sm:mt-0 mt-6";
+
+  // Title & description styles
+  const titleBaseClasses: string =
+    "lg:text-8xl text-6xl font-noto text-white font-bold max-w-xl h-auto mb-20";
+
+  const titleHomeClasses: string =
+    "lg:text-[150px] md:text-9xl sm:text-8xl text-6xl font-noto text-white font-bold max-w-7xl h-auto mb-8";
+
+  const paraHomeClasses: string =
+    "font-openSan md:text-2xl text-lg font-semibold text-white max-w-4xl h-auto mb-11 mt-40";
+
+  const paraBaseClasses: string =
+    "font-openSan text-2xl font-semibold text-white w-40 h-14 lg:mr-56 mr-20";
+
+  const wrapperHomeClass: string = "flex flex-row justify-between w-auto";
+
+  const wrapperBaseClass: string = "flex sm:flex-row flex-col w-auto";
+
+  const containerHomeClass: string =
+    "flex flex-col-reverse justify-between lg:ml-36 mx-12 mt-28 w-auto h-auto";
+
+  const containerBaseClass: string = "lg:ml-36 mx-12 mt-40";
+
   return (
     <>
-      <div className="text-3xl w-screen h-screen flex items-center justify-center bg-yellow-300">
-        Home Section
+      <div
+        style={imgSlide}
+        className="flex flex-col justify-between duration-500 delay-75"
+      >
+        {/* Destination Info */}
+        <div
+          className={
+            currentIndex === 0 ? containerHomeClass : containerBaseClass
```

Comment:
Nit: Some duplicated code here checking if `currentIndex === 0` 

Just suggestions:

- What does currentIndex === 0 actually represent?
- Could it be a variable?
- Could you setup an object structure that made these conditional classes much easier to consume?

Reply:
Yea I see what you mean haha. I think I've figured out a way to fix this going off your suggestions but not sure If I made it more complicated than it should be but Ill let you be the judge haha.

So my answer to the suggestions:
- What does currentIndex === 0 actually represent?
- Could it be a variable?
So I kinda figured out these two using this:
`const isHomeSlide = currentIndex === 0;
`
by using this variable the Homepage can see whether the first slide is showing or not

- Could you setup an object structure that made these conditional classes much easier to consume?
and for this one I created my object:
`const classStyles = [
    {
      titleHome:
        "lg:text-[150px] md:text-9xl sm:text-8xl text-6xl font-noto text-white font-bold max-w-7xl h-auto mb-8",
      titleBase:
        "lg:text-8xl text-6xl font-noto text-white font-bold max-w-xl h-auto mb-20",
    },
    {
      paraHome:
        "font-openSan md:text-2xl text-lg font-semibold text-white max-w-4xl h-auto mb-11 mt-40",
      paraBase:
        "font-openSan text-2xl font-semibold text-white w-40 h-14 lg:mr-56 mr-20",
    },
    {
      wrapperHome: "flex-row justify-between w-auto",
      wrapperBase: "flex sm:flex-row flex-col w-auto",
    },
    {
      containerHome:
        "flex flex-col-reverse justify-between lg:ml-36 mx-12 mt-28 w-auto h-auto",
      containerBase: "lg:ml-36 mx-12 mt-40",
    },
  ];`

and used variables to apply the appropriate styles to the homepage like this:
`const titleStyle = isHomeSlide
    ? classStyles[0].titleHome
    : classStyles[0].titleBase;`